src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformation.java

```diff
@@ -48,6 +58,29 @@ public ReleaseCreationInformation(
         this.pegoutCreationRskTxHash = pegoutCreationRskTxHash;
         this.pegoutBtcTx = pegoutBtcTx;
         this.pegoutConfirmationRskTxHash = pegoutConfirmationRskTxHash;
+
+        this.decodeUtxoOutpointValues(transactionReceipt);
+    }
+
+    private void decodeUtxoOutpointValues(TransactionReceipt transactionReceipt) {
+        List<LogInfo> logs =  transactionReceipt.getLogInfoList();
+
+        Function pegoutTransactionCreatedEvent = BridgeEvents.PEGOUT_TRANSACTION_CREATED.getEvent();
+        final byte[] pegoutTransactionCreatedSignatureTopic = pegoutTransactionCreatedEvent.encodeSignatureLong();
+
+        Optional<LogInfo> pegoutTransactionCreatedLog = logs.stream().filter((log ->
+            !log.getTopics().isEmpty() && Arrays.equals(log.getTopics().get(0).getData(),
+                pegoutTransactionCreatedSignatureTopic)
+        )).findFirst();
```

Comment:
No need for the extra parenthesis
```suggestion
        Optional<LogInfo> pegoutTransactionCreatedLog = logs.stream().filter(log ->
            !log.getTopics().isEmpty() && Arrays.equals(log.getTopics().get(0).getData(),
                pegoutTransactionCreatedSignatureTopic)
        ).findFirst();
```

Reply:
Solved.